3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -343,11 +361,23 @@ inline int ovx_hal_mul(const T *a, size_t astep, const T *b, size_t bstep, T *c,
 {
     try
     {
+        int rounding_policy = VX_ROUND_POLICY_TO_ZERO;
+        if ((float)scale != 0x1.010102p-8)
```

Comment:
OpenVX standard uses 0x1.010102p-8, so my suggestion is to force calculations to this exact number:
```
float fscale = (float)scale;
if (fabs(scale - 1/255.0f) < FLT_EPSILON)
    fscale = 0x1.010102p-8;
... use fscale only ...
```